WooCommerce/WooCommerceTests/ViewRelated/Coupons/CouponRestrictionsViewModelTests.swift

```diff
@@ -68,4 +68,44 @@ final class CouponRestrictionsViewModelTests: XCTestCase {
         let expectedTitle = String.localizedStringWithFormat(NSLocalizedString("Exclude Product Categories (%1$d)", comment: ""), 2)
         XCTAssertEqual(viewModel.excludeCategoriesButtonTitle, expectedTitle)
     }
+
+    func test_shouldDisplayLimitUsageToXItemsRow_is_false_when_fixed_cart_discount_type_is_set() {
+        let sampleCoupon = Coupon.fake().copy(discountType: .fixedCart)
```

Comment:
Nit: We should add clear comments for the condition and expectation of each test following Given/When/Then pattern.
```suggestion
        // Given
        let sampleCoupon = Coupon.fake().copy(discountType: .fixedCart)
```